configs/nginx/l2-explorer.conf.template

```diff
@@ -0,0 +1,29 @@
+server {
+    listen 443 ssl;
+    server_name explorer.${CERTBOT_DOMAIN_SUFFIX};
+
+    ssl_certificate /etc/letsencrypt/live/${CERTBOT_DOMAIN_SUFFIX}/fullchain.pem;
```

Comment:
@parketh jfyi, the cert won't be called bridge.tohma.snapchain.dev, explorer.tohma.snapchain.dev

now all certs are created under the same dir

```
# 2. obtain the SSL certificate for each subdomain
# the certs will be stored in /etc/letsencrypt/live/${CERTBOT_DOMAIN_SUFFIX}
# 
# note that Certbot creates a single certificate that's valid for all those 
# domains (called a SAN - Subject Alternative Names certificate)
# 
# after running the command, you can verify by:
#   sudo openssl x509 -in /etc/letsencrypt/live/${CERTBOT_DOMAIN_SUFFIX}/fullchain.pem -text | grep DNS:
# 
# reference: https://eff-certbot.readthedocs.io/en/latest/using.html
certbot certonly --nginx --non-interactive --agree-tos -m ${CERTBOT_EMAIL} \
  --cert-name ${CERTBOT_DOMAIN_SUFFIX} \
  -d rpc.${CERTBOT_DOMAIN_SUFFIX} \
  -d bridge.${CERTBOT_DOMAIN_SUFFIX} \
  -d explorer.${CERTBOT_DOMAIN_SUFFIX}
```

Reply:
so do we generate one certificate for the domain, and reuse it across tohma-devnet and tohma-finality-system servers? or are these two different certificates for the same domain?

Reply:
yes there is only one file

```
sudo ls /etc/letsencrypt/live/tohma.snapchain.dev                                                                                 ─╯
README	cert.pem  chain.pem  fullchain.pem  privkey.pem
```

and you can run this command on the server to test

```
sudo openssl x509 -in /etc/letsencrypt/live/tohma.snapchain.dev/fullchain.pem -text | grep DNS:                                   ─╯
                DNS:bridge.tohma.snapchain.dev, DNS:explorer.tohma.snapchain.dev, DNS:rpc.tohma.snapchain.dev
```

Reply:
it's a single cert the valid for all domains